packages/calcite-components/src/assets/styles/_floating-ui.scss

```diff
@@ -1,17 +1,19 @@
+$floating-ui-transition-offset: 5px;
```

Comment:
@alisonailea I wonder if we should have this as some kind of global token instead of a Sass var?

Reply:
Or maybe one of these could be used?

```
  --calcite-spacing-xs: 6px;
  --calcite-spacing-xxs: 4px;
```

```
--calcite-size-xs: 6px;
  --calcite-size-xxs: 4px;
```